libs/common/src/interfaces/dsnp.ts

```diff
@@ -0,0 +1,183 @@
+/**
```

Comment:
this can deleted if we are able to load activity-content but not needed for this PR work